llvm/lib/Target/Hexagon/HexagonBitSimplify.cpp

```diff
@@ -2957,8 +2942,8 @@ namespace {
 
 char HexagonLoopRescheduling::ID = 0;
 
-INITIALIZE_PASS(HexagonLoopRescheduling, "hexagon-loop-resched",
-  "Hexagon Loop Rescheduling", false, false)
+INITIALIZE_PASS(HexagonLoopRescheduling, "hexagon-loop-resched-pass",
```

Comment:
renaming the pass as it conflicts with a cl option of the same name. It seems pass initialization creates a cl opt with the same name as the pass. 

This conflict did not happen in earlier code because not all passes where initialized during target init, some were on demand so that ended up hiding the conflict.